frontend/public/locales/en/new-application.json

```diff
@@ -29,5 +29,18 @@
       "paragraph-one": "By starting this application, you consent to the terms of the above privacy notice statement.",
       "button": "Start application"
     }
+  },
+  "a2": {
+    "page-title": "Type of application",
+    "breadcrumbs": {
+      "new-application": "New Application",
+      "a2": "Type of Application"
```

Comment:
**Nitpick:** `a2` could be named something a little more descriptive, in my opinion.

Reply:
Updated.